DigitalLearningSolutions.Data.Tests/DataServices/PasswordDataServiceTests.cs

```diff
@@ -14,9 +14,9 @@
     public class PasswordDataServiceTests
     {
         private const string PasswordHashNotYetInDb = "I haven't used this password before!";
+        private SqlConnection connection = null!;
         private PasswordDataService passwordDataService = null!;
         private UserDataService userDataService = null!;
-        private SqlConnection connection = null!;
 
         [SetUp]

```

Comment:
There are a number of tests in this file that don't really make sense anymore. For instance, we have a few tests for `SetPasswordForUsersAsync` that supposedly check various accounts have their password set, but these effectively duplicate each other as they are all checking the same User record instead of AdminAccount/DelegateUsers. 

Again, we are in a weird place with regards to what we want to refactor right this second, so exactly how much of these tests we want to change is questionable. But I think we can safely remove a fair few of these. Might require a deeper think though.

Reply:
I think I removed all the ones that we don't need anymore